yarn-project/yarn.lock

```diff
@@ -21994,25 +21938,24 @@ __metadata:
   languageName: node
   linkType: hard
 
-"viem@npm:2.22.8":
-  version: 2.22.8
-  resolution: "viem@npm:2.22.8"
+"viem@npm:2.23.7":
```

Comment:
Heads up we also got a viem 2.23.5 coming from somewhere according to this yarn.lock (see below this viem entry)